forch/forch_metrics.py

```diff
@@ -32,6 +41,19 @@ def start(self):
         finally:
             threading.Thread(target=self._http_server.start_server(), daemon=True).start()
 
+        self._register_metric_pages()
+
+        self._proxy_server = forch.http_server.HttpServer(self._proxy_port)
+        try:
+            for page in self._metric_pages:
+                self._proxy_server.map_request(
+                    page, lambda path, params: self._get_path_metrics(path.split('/')[1]))
```

Comment:
what about the rest of the path?  Shouldn't this just be everything except the first bit?

Reply:
I didnt get it. Rest of the path?

Reply:
You take the input path, split it by /, and then only take [1] -- what about [2].. [n]?

Reply:
Addressed.